doc/distrib/NodeHelpFiles/_review/Revit.Elements.Room.Location.md

```diff
@@ -0,0 +1,10 @@
+## In Depth
+`Room.Location` returns a Dynamo point that represents a given room's location.
```

Comment:
It would be good to point out where is this point located to provide context for users. Suggest adding "The location point of a particular room is on it's floor and at the intersection of the lines of the cross displayed by Revit when the room is selected. " Another useful note here can be: "You can determine its elevation by checking the Z coordinate of the point."